website_sale_ajaxify_cart/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "eCommerce Ajaxify cart",
+    "summary": "eCommerce Ajaxify cartr",
```

Comment:
As a human-readable name this is a bit cryptic. You might consider to improve it.